<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CardLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Please enter account number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Please enter password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pinButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay it Now!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lmt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Let me Think</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Holder Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="namelineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Please enter the card holder's name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount_F">
    <property name="title">
     <string>Account(&amp;F)</string>
    </property>
   </widget>
   <addaction name="menuAccount_F"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lmt</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pinButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
